e2e/support/helpers/e2e-viz-settings-helpers.js

```diff
@@ -6,8 +6,14 @@ export function openSeriesSettings(field, isBreakout = false) {
       .find(".Icon-ellipsis")
       .click();
   } else {
-    cy.findAllByTestId("chartsettings-field-picker")
-      .contains(field)
+    cy.findAllByTestId("chartsettings-field-picker-select")
+      .then($elements => {
+        for (const element of $elements) {
+          if (element.value === field) {
+            return cy.wrap(element);
+          }
+        }
+      })
```

Comment:
I can't do this because cypress gives a jquery array instead of an actual array - so I can't access values using the normal array methods without some other manipulations 🙃

Reply:
@oisincoveney oh right, I think `[...elements]` or `Array.from(elements)` _might_ work though but I never tried that